Miningbase Rework

Lovebird-Frungy opened the pull request:
# General Documentation

### Intent of your Pull Request

Remakes the entire mining base in order to bring a better playing experience to mining, with (vacant) room(s) for improvement, adapting to further content.

Overall its been pretty fun to work on this project (this is the sixth time i do it all over again), and i’ll understand if people prefer to stick to the old one.

### Why is this change good for the game?

The changes are made to address two issues:
-The mining base being an experimental facility on a deadly planet, it needed to be tweaked a bit more and security amped up, that is where the reworked atmos, closer medical facility, blast doors and secondary exits come in, the base being often dented, it needs to be built to last, the current mining base does not have that much resilience.
-The mining base is the place some players will spend most of their shift, and as we’re shifting into a more rp server, i felt it was necessary to make it a better place to live in order to facilitate interactions between players, that's where the vacant room, closer break room, and reworked layout come in. You no longer have to go to the other end of the base to see if other miners are chilling.

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 

Miningbase has been revamped, layout completely redone and new areas added.
Now the areas are:
Two EVAS
Living area with internal shuttle dock
Engineering (Tcomms, atmos and power)
Vacant Room
Vacant Maintenance
Break Room
Infirmary
Processing (in case the orm blew up, a relic of times of yore that will probably be removed soon and then the room will be repurposed)

New mining base picture:
<img width="427" alt="NuMining_Love" src="https://user-images.githubusercontent.com/64857565/118308640-7c8f9c00-b4ec-11eb-8e9f-3a9f61011c31.png">

With areas:
<img width="425" alt="NuMining_Love_Areas" src="https://user-images.githubusercontent.com/64857565/118308689-8d401200-b4ec-11eb-9efe-a39d0175b36c.png">

New features:
Internal shuttle dock, pretty and safe, with railings to prevent accidental gibbing.
Infirmary directly linked to EVA.
A secondary EVA and an emergency exit.
Personal miner equipment lockers with gps.
Anchored lockers so stuff doesn’t fly out during breaches, making the place nicer to look overall.
Mining lock down system with blast doors
Bunk dormitories (overall the only downgrade of the pr, the dorms are not private anymore, but i feel it is fitting rp-wise as life ain’t cheap and this is a base of operation, not a luxury hotel)
Curtained windows to prevent local fauna from being agitated too much.
Reworked gas mining, fully functional and allowing mining to be self sufficient atmos-wise.
Security post.
Disposals leading to lava for trash disposing.
Cryopods.
Mining medic locker now has binoculars (in order to stay away from megafauna), and earmuffs (for gibtonite ear damage healing), probably should add this to the base locker content but might need some input.
Bigger external access for less clogging.

Mining shuttle has been remade in order to adapt to the new layout, all the while being compatible with other stations.

New mining shuttle picture:
![image](https://user-images.githubusercontent.com/64857565/116302234-3a065980-a7a1-11eb-8001-0d40b0e20b3a.png)

New area have been added, old areas renamed and new area sprites done for easier future mapping.

### What should players be aware of when it comes to the changes your PR is implementing?

They now have a locker where they can find their gps and stash their loot.
A portable air pump can be used to overcharge their oxygen tanks.
New vacant room for them to use their imagination.
Auxiliary EVA to facilitate work.

### What general grouping does this PR fall under? 

Mining Mapping.

### Are there any aspects of the PR that you would like us not to mention on the Wiki?

No.

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 

No.

# Changelog

:cl:  Lovebird
rscadd: Shiny new miningbase
rscdel: Removed the old miningbase
imageadd: Added a few area icons for mapping
/:cl:

Commit:
Mining Base Rework

Here is the core of the pr.
New miningbase layout
New mining shuttle layout
New areas and reworked area names
New area sprites

Commit:
Sign Fix

Fixes the mining signs (y:-32 to y:32)

Commit:
More fixes

Adds a few firealarms and fire extinguishers i missed.
Adds four windows, one in the processing roome, one in the bar, two in the vacant room. Other machinery have been moved accordingly.
The bookcase not working, replaced with a pajama wardrobe with plushies!
Opacifies the recovery room door for privacy.

Commit:
Adds some decoration, vents and rwalls.

Adds one vent and one scrubber.
Adds posters and a few new newcasters to the base and the shuttle.
Makes the tcomms walls reinforced like the old base.

Commit:
Portable Scrubber!

Adds a portable scrubber to the atmos room!

patpol4 committed:
fixes and tables

patpol4 reviewed (changes requested):
Fixed some things for you.

Added Frank
Added tables back to shuttle, 
added more lights
![dreamseeker_tuH4eaAGRr](https://user-images.githubusercontent.com/48154165/117631010-174e4a80-b17c-11eb-875e-e0bd6f6eab96.png)
made morgue have a bit more space
![vivaldi_9P0UzO7zdF](https://user-images.githubusercontent.com/48154165/117631055-22a17600-b17c-11eb-89dc-626d3867aef7.png)
Fixed some active turfs, 
fixed the apc and airlock controller offsets
added a lot of decals to make it a bit less bland
changed some pipes to be visible or invisible
added some crates
fixed this stinky error
![vivaldi_W6mPGj9d3d](https://user-images.githubusercontent.com/48154165/117631159-3e0c8100-b17c-11eb-8a76-186ff7f6899c.png)

Anyway, I'll testmerge this but there is still some work to be done, mainly on this room, 
![dreamseeker_6GNSzALB7J](https://user-images.githubusercontent.com/48154165/117631445-8af05780-b17c-11eb-81de-465e2154cf3f.png)
its too clean for an abandoned one, if you want it to be sort of an empty room then just block it off with a barricade and add dirt and broken stuff inside, if not then do something else with it, it just can't stay as it is

Commit:
Merge remote-tracking branch 'origin/Miningbase_Rework_Lovebird' into Miningbase_Rework_Lovebird

Commit:
Minor Changes

Adds a random book to the recovery room.
Adds some dirt and oil to processing.
Adds a medical mask to go with the anesthetics tank.

Commit:
Minor Tweaks

Adds three holopads (Lobby, Break Room, Infirmary)
Adds a book to the dorms.
Changes the mining medic locker contents themselves, additionally adding a crew pinpointer.
Moves the shuttle poster.
Moves wall mounts around.
Moves the morgue window to make it accessible in order to interact with its curtains.
Opacifies the morgue window.
Changes the vacant room door to engineering colors.

Commit:
Reverts

Reverts some stuff, to start over again, hope this works.

Commit:
Cosmetic Changes

Changed some decals.

Commit:
Maint Door Change

Adds the abandoned helper to the forgotten door.

Commit:
Closet: the lockerening.

As per feedback, upgraded the closets to their own kind of personal closet.
Secwear vendor changed into secure contraband locker.

Commit:
Closetor IV: Retribution (Part 3 of the war of the decalums decalogy)

Adds (src) to the new crate code’s ( as per SomeguyManperson’s advice, thanks to them, this pr is less of a mess!).
Adds some decals (deathsposal caution, portable atmos placement, and smes floor).

Commit:
First Aid 1.23: Life, Death, Tea and Biscuits.

Adds an emergency medical crate to the infirmary. 
Containing a first aid kit, a toxin kit and an oxyloss kit.
Also swaps an ore box for symetry.

patpol4 reviewed (approved):
After months of testmerge i believe this is good to go, will merge later today.
If you have any last comments, now's the time.
